src/pages/bridge/Issue/index.tsx

```diff
@@ -53,8 +66,9 @@ function Issue(props: IssueProps): JSX.Element {
 
   const maxIssuable = nativeToDecimal(issuableTokens || 0).toNumber();
 
-  const { handleSubmit, watch, register, formState, setValue } = useForm<IssueFormValues>({
-    resolver: yupResolver(getIssueValidationSchema(maxIssuable, parseFloat(balance || '0.0'))),
+  const { handleSubmit, watch, register, formState, setValue, trigger } = useForm<IssueFormValues>({
+    resolver: yupResolver(getIssueValidationSchema(maxIssuable, parseFloat(balance || '0.0'), tokenSymbol)),
+    mode: 'onBlur',
```

Comment:
I think `onBlur` is not often enough. Let's trigger the validation as often as possible, ie every time the input changes. Otherwise, I see errors although they should be gone already.

<img width="1671" alt="image" src="https://github.com/pendulum-chain/portal/assets/6690623/25eb3002-8013-4ab1-b9ba-b2e91c1d9288">